### PR TITLE
Exclude interfaces from code climate

### PR DESCRIPTION
**Pull request by sansive**
## Task
Exclude interfaces from code climate

### CODE REVIEW

**Review by Oxydeas — APPROVED** [review]

**Review by Geno22 — APPROVED** [review]
Great let's hope it bumps up the coverage !

File: app/src/main/java/com/github/houseorganizer/houseorganizer/panels/login/RegisterEmail.java [hunk 1]
```diff
@@ -62,11 +62,18 @@ protected void onCreate(Bundle savedInstanceState) {
         );
     }
 
-
+    private void initializeVariables() {
+        mAuth = FirebaseAuth.getInstance();
+        isEmailAlreadyUsed = false;
+        email = ((EditText) findViewById(R.id.reg_enter_email)).getText().toString();
+        password = ((EditText) findViewById(R.id.reg_enter_password)).getText().toString();
+        confPassword = ((EditText) findViewById(R.id.reg_confirm_password)).getText().toString();
```
Comment (Geno22) [inline app/src/main/java/com/github/houseorganizer/houseorganizer/panels/login/RegisterEmail.java:70]: I I think the elements that are in the fields are supposed to be retrieved only when clicking the button, here you take them at the beginning when they are empty I think

**Review by Geno22 — APPROVED** [review]
Thanks !

**Comment by sansive** [comment]
@Oxydeas I pushed some new changes if you want to take a look at them before merging

**Review by Geno22 — COMMENTED** [review]
According to codeclimate the total coverage is still 64%...
I don't get where it comes from

**Review by Oxydeas — APPROVED** [review]

**Comment by sansive** [comment]
> According to codeclimate the total coverage is still 64%... I don't get where it comes from

Yes, that's true. I have posted a message on Github discussions to see if we can get any help

**Pull request merged**